packages/directlinespeech/__tests__/sendSpeechActivity.js

```diff
@@ -62,7 +84,7 @@ test('should echo back "Bellevue" when saying "bellview"', async () => {
 
 // 2020-05-11: Direct Line Speech protocol was updated to synthesize "text" if "speak" property is not set.
 test('should synthesis if "speak" is empty', async () => {
-  const { directLine, sendTextAsSpeech } = await createTestHarness();
+  const { directLine, sendTextAsSpeech } = await createTestHarness({ enableInternalHTTPSupport: false });
```

Comment:
If we set the default value for the options parameter to an empty object in the function definition, then we don't need to pass the options as a parameter to `createTestHarness` and `enableInternalHTTPSupport` will inherently be falsy. 

```suggestion
  const { directLine, sendTextAsSpeech } = await createTestHarness();
```